sources/packages/web/src/services/http/dto/Note.dto.ts

```diff
@@ -8,6 +8,11 @@ export interface NoteAPIInDTO {
   description: string;
 }
 
+export interface NoteTypeItems {
```

Comment:
It seems to be an interface to be reused by vuetify components. Can we create a generic interface under `sources\packages\web\src\types\vuetify.ts`?